.github/workflows/architecture-validation.yml

```
@@ -81,41 +81,47 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v7
 
-    - name: Check domain → analysis violations
+    - name: Enforce crate layering
       run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1087:error:32:27: Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet)

(shellcheck)